### PR TITLE
open-cluster-management/backlog#6105 --pattern started WIP

### PR DESCRIPTION
**Pull request by swopebe**

### CODE REVIEW

File: manage_applications/manage_apps_git.adoc [hunk 1]
```diff
@@ -88,3 +88,57 @@ data:
...
+
+A Git repository to store this type of information can resemble the following repositoty list:
+
+----
+(MAIN branch) rhacm-hub-cluster/
+  common-managed-apps/
+    app-name-0/
+      application.yaml
+      subscription.yaml
+      channel.yaml     # This points to `app-name-0` manifest repo, which contains `Deployable` manifests, such as `deployment.yaml` or Helm chart
```
Comment (jnpacker) [inline manage_applications/manage_apps_git.adoc]: take out the word `Deployable`.

File: manage_applications/manage_apps_git.adoc [hunk 2]
```diff
@@ -88,3 +88,57 @@ data:
...
+      placementrule.yaml
+    app-name-1/
+      application.yaml
+      subscription.yaml
+      channel.yaml    # This points to `app-name-1` manifest repo, which contains `Deployable` manifests, such as `deployment.yaml` or Helm chart
```
Comment (jnpacker) [inline manage_applications/manage_apps_git.adoc]: Remove `Deployable`

File: manage_applications/manage_apps_git.adoc [hunk 3]
```diff
@@ -88,3 +88,60 @@ data:
...
+
+In the following example, there is a folder for each hub cluster. You can also create a repository for each hub cluster or a branch for each hub cluster. 
+
+A single subscription defined on the hub cluster pulls in all the other configuration resources, such as hub cluster config, policies, and common applications that will configure and deploy to the hub and managed clusters.
+
+A Git repository to store this type of information can resemble the following repositoty list:
+
+----
+(MAIN branch) rhacm-hub-cluster/
+  common-managed/
```
Comment (swopebe) [inline manage_applications/manage_apps_git.adoc]: Make that a variable.

Comment (swopebe) [reply manage_applications/manage_apps_git.adoc]: Call this best practice.

**Pull request merged**